vanilla-framework/vanilla-framework
# 1503: Fix navigation breakpoint usage

## Done

- Fixed a bug in the navigation pattern where it wasn't using the navigation breakpoint variable

## QA

- Pull code
- Open the file at `/vanilla-framework/scss/_settings_breakpoints.scss`
- Edit the `$breakpoint-navigation-threshold` to anything else
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/navigation/default/
- Check that the navigation changes properly at the new breakpoint you set (make sure to click on Menu for small screens)

## Details

Fixes #1503 

## Screenshots

### Before:
![screenshot 7](https://user-images.githubusercontent.com/25733845/35328242-98746424-00f3-11e8-9900-542233ddcb65.png)


### After:
![screenshot 6](https://user-images.githubusercontent.com/25733845/35328200-78717644-00f3-11e8-99ce-2256288c2983.png)
